### PR TITLE
PPTP-278: initial start page

### PR DESCRIPTION
**Pull request by tombart**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -106,4 +106,10 @@ assets {
   url = "http://localhost:9032/assets/"
 }
 
+urls {
```
Comment (PauloPortugal) [inline conf/application.conf:109]: Do we need these links?
If these are Start page URLs, then we should probably treat them as content.

File: conf/messages.en [hunk 2]
```diff
@@ -4,4 +4,32 @@ feedback.link=feedback
 service.name=Plastic Packaging Tax
 
 title.format={0} - {1} - GOV.UK
-title.withSection.format={0} - {1} - {2} - GOV.UK
\ No newline at end of file
+title.withSection.format={0} - {1} - {2} - GOV.UK
+
```
Comment (PauloPortugal) [inline conf/messages.en:8]: :+1: Looks good

File: app/uk/gov/hmrc/plasticpackagingtax/registration/config/ExternalServicesConfig.scala [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+@Singleton
+class ExternalServicesConfig @Inject()(val configuration: Configuration) {
+
+  private def loadUrl(key: String): String =
+    configuration.getOptional[String](s"urls.$key").getOrElse(throw new Exception(s"Missing configuration key: urls.$key"))
+
+  val serviceAvailabilityUrl: String = loadUrl("serviceAvailability")
+  val govUkUrl: String = loadUrl("govUk")
+  val organisationsUrl: String = loadUrl("organisationsLink")
+  val explorePpt: String = loadUrl("explorePpt")
```
Comment (PauloPortugal) [inline app/uk/gov/hmrc/plasticpackagingtax/registration/config/ExternalServicesConfig.scala:31]: Do we need all these urls?

File: conf/app.routes [hunk 4]
```diff
@@ -2,3 +2,4 @@
 
 GET        /assets/*file            controllers.Assets.versioned(path="/public", file: Asset)
 GET        /hello-world             uk.gov.hmrc.plasticpackagingtax.registration.controllers.HelloWorldController.helloWorld
```
Comment (PauloPortugal) [inline conf/app.routes:4]: Once the start page is in we should be able to remove the `/hello-world` URI

File: test/Stubs.scala [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+      |metrics.logback=false
+      |draft.timeToLive=1d
+      |timeoutDialog.timeout=13min
+      |timeoutDialog.countdown=3min
+      |list-of-available-journeys = "SMP,STD,CAN,SUB,CON"
+      |microservice.services.features.default=disabled
+      |microservice.services.features.use-improved-error-messages=true
+      |urls.tradeTariff=tradeTariff
+      |urls.classificationHelp=classificationHelp
+      |urls.ecicsTool=ecicsTool
```
Comment (PauloPortugal) [inline test/Stubs.scala:76]: We should not need these Config options, as these are just specific to Exports

**Review by PauloPortugal — COMMENTED** [review]
Looks good!
Please address the comments and remove urls and configs that are not required by the project.

File: conf/application.conf (hunk 1 above)
Comment (tombart) [reply conf/application.conf:109]: ok, will move them to content

File: test/Stubs.scala (hunk 5 above)
Comment (tombart) [reply test/Stubs.scala]: removed

File: conf/application.conf (hunk 1 above)
Comment (tombart) [reply conf/application.conf]: done

File: app/uk/gov/hmrc/plasticpackagingtax/registration/config/ExternalServicesConfig.scala (hunk 3 above)
Comment (tombart) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/config/ExternalServicesConfig.scala]: removed govUk as not needed, the other ones I guess they will be needed at some point, serviceAvailability probably will be included on the page, explorePpt is for info on PP tax, organisationsLink is general for hmrc, can removed it for now if not required on start page, what do you think?

File: conf/app.routes (hunk 4 above)
Comment (tombart) [reply conf/app.routes]: ok, I'm currently pointing my start button to hello world controller, thought I'll leave that around until we have next page to go to. Do you want  me to remove that and just leave blank button url?

File: app/uk/gov/hmrc/plasticpackagingtax/registration/config/ExternalServicesConfig.scala (hunk 3 above)
Comment (tombart) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/config/ExternalServicesConfig.scala]: removed

File: conf/app.routes (hunk 4 above)
Comment (tombart) [reply conf/app.routes]: as agreed, removed

**Review by PauloPortugal — APPROVED** [review]

**Pull request merged**